conanfile.py

```diff
@@ -0,0 +1,97 @@
+import os
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+from conan.tools.scm import Git, Version
+from conan.tools.files import update_conandata, download, unzip
+
+required_conan_version = ">=2.0"
+
+class DPPConan(ConanFile):
+    name = "dpp"
+    version = "10.0.33"
+    default_user = "brainboxdotcc"
+    default_channel = "testing"
```

Comment:
Default user and channel do not have any effect in Conan 2.
The way to hardcode it in Conan 2 is using ``user="xxx"`` and ``channel="yyyy"``. But in many cases, specially opensource like ConanCenter, it is fine to go without user/channel.